plumbing/api.go

```diff
@@ -65,7 +65,7 @@ type API struct {
 type APIDeps struct {
 	Bitswap       exchange.Interface
 	Chain         *cst.ChainStateProvider
-	ChnState      *consensus.ChainState
+	ChnState      *consensus.ActorState
```

Comment:
nit: this should probably be renamed ActorState on the api

Reply:
Done.

Reply:
Or use a name like `ChainState` for the object.